### PR TITLE
Remove carbon term from packages

### PR DESCRIPTION
**Pull request by erandacr**

### CODE REVIEW

File: components/org.wso2.transport.jms/pom.xml [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
+        <artifactId>jms-parent</artifactId>
         <version>6.0.48-SNAPSHOT</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <packaging>bundle</packaging>
-    <name>WSO2 Carbon Transport JMS Component</name>
-    <artifactId>org.wso2.carbon.transport.jms</artifactId>
+    <name>WSO2 Transport JMS Component</name>
+    <artifactId>org.wso2.transport.jms</artifactId>
```
Comment (gokul) [inline components/org.wso2.transport.jms/pom.xml]: Are groupID and artifactID the same for the component? (i.e. org.wso2.transport.jms)

Comment (erandacr) [reply components/org.wso2.transport.jms/pom.xml]: yes, as we were discussed I had to change group id to "org.wso2.transport.jms", after removing the carbon term I ended up artifact id as the same.

**Review by gokul — APPROVED** [review]

File: components/org.wso2.transport.jms/pom.xml (hunk 1 above)
Comment (bsenduran) [reply components/org.wso2.transport.jms/pom.xml]: what about `transport-jms` as artifact id ?

Comment (bsenduran) [reply components/org.wso2.transport.jms/pom.xml]: FYI : https://maven.apache.org/guides/mini/guide-naming-conventions.html

Comment (erandacr) [reply components/org.wso2.transport.jms/pom.xml]: I haven't changed that as we are still under discussion

Comment (erandacr) [reply components/org.wso2.transport.jms/pom.xml]: Updated artifact ids

**Review by VijithaEkanayake-zz — APPROVED** [review]

**Pull request merged**